app/src/ui/check-runs/ci-check-run-popover.tsx

```diff
@@ -265,7 +269,20 @@ export class CICheckRunPopover extends React.PureComponent<
       }
     }
 
-    return <Donut valueMap={getCheckStatusCountMap(checkRuns)} />
+    const valueMap = getCheckStatusCountMap(checkRuns)
+
+    const ariaLabel = `Completeness indicator. ${
+      valueMap.get(APICheckStatus.Completed) ?? 0
+    } completed, ${valueMap.get(APICheckStatus.InProgress) ?? 0} in progress, ${
+      valueMap.get(APICheckStatus.Queued) ?? 0
+    } queued.`
+
+    return (
+      <Donut
+        ariaLabel={ariaLabel}
+        valueMap={getCheckStatusCountMap(checkRuns)}
```

Comment:
We can use the variable above right?

```suggestion
        valueMap={valueMap}
```

Reply:
Haha! Yes. I was too hyper focused on the ariaLabel.